module/VuFind/src/VuFind/Search/ProQuestFSG/Options.php

```diff
@@ -56,48 +52,15 @@ public function __construct(\VuFind\Config\PluginManager $configLoader)
         $this->searchIni = $this->facetsIni = 'ProQuestFSG';
         parent::__construct($configLoader);
 
-        // Load the configuration file:
-        $searchSettings = $configLoader->get($this->searchIni);
-
-        // Set up limit preferences
-        if (isset($searchSettings->General->default_limit)) {
-            $this->defaultLimit = $searchSettings->General->default_limit;
-        }
-        if (isset($searchSettings->General->limit_options)) {
-            $this->limitOptions = $this->explodeListSetting($searchSettings->General->limit_options);
-        }
-        if (isset($searchSettings->General->result_limit)) {
-            $this->resultLimit = min(intval($searchSettings->General->result_limit), 1000);
-        } else {
+        // Set result limit to a value we can support:
+        if (-1 === $this->resultLimit) {
```

Comment:
If you decide to introduce defaultResultLimit, you could simplify this code too.

Reply:
Done.